src/Symfony/Bundle/FrameworkBundle/Resources/config/workflow.xml

```diff
@@ -4,6 +4,10 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="workflow.guard_listener">Symfony\Component\Workflow\EventListener\GuardListener</parameter>
```

Comment:
we try to avoid using parameters when not needed, looks the case here

Reply:
Ok :)